hydra_base/db/model.py

```diff
@@ -467,9 +468,10 @@ class TemplateType(Base, Inspect):
     )
 
     id = Column(Integer(), primary_key=True, nullable=False)
-    name = Column(String(60),  nullable=False)
+    name = Column(String(200),  nullable=False)
+    description = Column(String(1000))
```

Comment:
Is 1000 characters enough for a description?

Reply:
Possibly, but this would be the focus of another piece of work